### DISCUSSION TITLE
Shared directory from macOS to Ubuntu not working

### DISCUSSION THREAD

**Discussion by LLuth0r** [cirruslabs/tart#707] [Q&A]
Hi!  I have tried running `tart run --dir=project:~/Code/project ubuntu` but I can't seem to find the shared directory from my macOS (Sonoma) host in my Ubuntu VM.  Is this command wrong or am I looking for it in the wrong location?

**Comment by edigaryev** [comment]
Directory mounting inside of Linux guests is not automatic (unless you configure it as so in e.g. `/etc/fstab`).

>Is this command wrong or am I looking for it in the wrong location?

Please take a look at [Accessing mounted directories in Linux guests](https://tart.run/quick-start/#accessing-mounted-directories-in-linux-guests).

**Reply by LLuth0r** [reply]
Thank you!  I had to create the "shared" directory, but after that it worked.
